tiled/README.md

```diff
@@ -83,6 +83,17 @@ a certain function on them.
 
 `isEmpty` and `isNotEmpty` extension method to check if the specific collection is empty or not.
 
+### `BatchTiledMapRenderer`
+
+`use` extension method to call `beginRender()` and `endRender()` automatically before
+your render logic.
+
+**Careful**: Since `beginRender()` and `endRender()` are protected methods, they are
```

Comment:
Is this still relevant? I think this section can be removed.

Reply:
No, that can be removed. I forgot that, sorry